packages/frontend/editor-ui/src/features/dataStore/components/dataGrid/DataStoreTable.vue

```diff
@@ -226,6 +228,12 @@ const onDeleteColumn = async (columnId: string) => {
 			props.dataStore.projectId,
 			columnId,
 		);
+		telemetry.track('User deleted data table column', {
+			column_id: columnId,
+			column_name: columnToDelete.headerName,
```

Comment:
I see that we've dropped the column names from some of the other track calls, so just make sure this one and the next are not forgotten.

Reply:
Yeah, waiting for response